client/index.html

```diff
@@ -120,7 +120,7 @@ <h1 class="title">Connect</h1>
 								<div class="col-sm-3"></div>
 								<div class="col-sm-9">
 									<label class="tls">
-										<input type="checkbox" name="tls" <%= defaults.tls ? 'checked="checked"' : '' %>>
+										<input type="checkbox" name="tls" <%= defaults.tls ? 'checked="checked"' : '' %> <%= typeof(lockNetwork) !== "undefined" && lockNetwork ? 'disabled' : ''%>>
```

Comment:
This seems reasonable to me, to avoid the annoying escaping of the double-quotes for the HTML. This is consistent with what's already there. Double-quotes for Javascript expressions, single quotes for the HTML output.